cobra/README.md

```diff
@@ -17,13 +17,12 @@ the right structure so you can immediately enjoy all the benefits of Cobra. It
 will also automatically apply the license you specify to your application.
 
 Cobra init is pretty smart. You can either run it in your current application directory
-or you can specify a path to an existing project.  If the directory you point it at does not
-exist, it will be created for you.
+or you can specify a relative path to an existing project.  If the directory does not exist, it will be created for you.
```

Comment:
Is the double space before 'If' on purpose?

Reply:
nope.  thanks :)